Microsoft.Toolkit.Uwp.UI.Controls.Input/RichSuggestBox/RichSuggestBox.Events.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// The RichSuggestBox control extends <see cref="RichEditBox"/> control that suggests and embeds custom data in a rich document.
+    /// </summary>
+    public partial class RichSuggestBox
+    {
+        /// <summary>
+        /// Event raised when the control needs to show suggestions.
+        /// </summary>
+        public event TypedEventHandler<RichSuggestBox, SuggestionsRequestedEventArgs> SuggestionsRequested;
+
+        /// <summary>
+        /// Event raised when user click on a suggestion.
+        /// This event lets you customize the token appearance in the document.
+        /// </summary>
+        public event TypedEventHandler<RichSuggestBox, SuggestionChosenEventArgs> SuggestionChosen;
+
+        /// <summary>
+        /// Event raised when a token is fully highlighted.
+        /// </summary>
+        public event TypedEventHandler<RichSuggestBox, RichSuggestTokenSelectedEventArgs> TokenSelected;
+
+        /// <summary>
+        /// Event raised when a pointer is hovering over a token.
+        /// </summary>
+        public event TypedEventHandler<RichSuggestBox, RichSuggestTokenHoveringEventArgs> TokenHovering;
```

Comment:
As yes, this name was intended to be different from `PointerOver` since it worked a bit differently when I first created it. Now it essentially works the same way so I will be changing the name to `TokenPointerOver`. 👍

Reply:
`TokenPointerOver` makes sense to me. Changing now.